[2022-01-31 17:23:39,964] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest manual__2022-01-31T17:23:32.073582+00:00 [queued]>
[2022-01-31 17:23:40,075] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest manual__2022-01-31T17:23:32.073582+00:00 [queued]>
[2022-01-31 17:23:40,078] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 17:23:40,081] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-31 17:23:40,083] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 17:23:40,193] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-01-31 17:23:32.073582+00:00
[2022-01-31 17:23:40,223] {standard_task_runner.py:52} INFO - Started process 39439 to run task
[2022-01-31 17:23:40,252] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'testing_dag', 'ingest', 'manual__2022-01-31T17:23:32.073582+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmp2grb4ud1', '--error-file', '/tmp/tmp1_rkxmt1']
[2022-01-31 17:23:40,260] {standard_task_runner.py:77} INFO - Job 122: Subtask ingest
[2022-01-31 17:23:40,846] {logging_mixin.py:109} INFO - Running <TaskInstance: testing_dag.ingest manual__2022-01-31T17:23:32.073582+00:00 [running]> on host 5beebf73c64a
[2022-01-31 17:23:41,711] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T17:23:32.073582+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-31T17:23:32.073582+00:00
[2022-01-31 17:23:41,739] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-31 17:23:41,759] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-01-31 17:23:41,839] {subprocess.py:85} INFO - Output:
[2022-01-31 17:23:42,131] {subprocess.py:89} INFO - ***-worker.pid
[2022-01-31 17:23:42,134] {subprocess.py:89} INFO - ***.cfg
[2022-01-31 17:23:42,138] {subprocess.py:89} INFO - dags
[2022-01-31 17:23:42,150] {subprocess.py:89} INFO - google_credentials.json
[2022-01-31 17:23:42,155] {subprocess.py:89} INFO - logs
[2022-01-31 17:23:42,164] {subprocess.py:89} INFO - output.csv
[2022-01-31 17:23:42,183] {subprocess.py:89} INFO - output_2021-01.csv
[2022-01-31 17:23:42,187] {subprocess.py:89} INFO - output_2021-02.csv
[2022-01-31 17:23:42,190] {subprocess.py:89} INFO - output_2021-03.csv
[2022-01-31 17:23:42,192] {subprocess.py:89} INFO - output_2021-04.csv
[2022-01-31 17:23:42,197] {subprocess.py:89} INFO - output_2022-01.csv
[2022-01-31 17:23:42,199] {subprocess.py:89} INFO - plugins
[2022-01-31 17:23:42,202] {subprocess.py:89} INFO - requirements.txt
[2022-01-31 17:23:42,205] {subprocess.py:89} INFO - webserver_config.py
[2022-01-31 17:23:42,208] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-31 17:23:42,780] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20220131T172332, start_date=20220131T172339, end_date=20220131T172342
[2022-01-31 17:23:43,748] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-31 17:23:44,444] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
